### ISSUE TITLE
Update the tutorial video, based on the latest SDK

### ISSUE THREAD

**Issue by shteryana** [Stremio/stremio-addon-sdk#68]

**Comment by jaruba** [comment]
Yeah, this is still on my to do list, I updated the code in the written tutorial for it (link is in the video description), but still need to update the video too, it may also require switching to the new igdb api if their module changed along with their new api
